android/java/org/chromium/chrome/browser/app/domain/NetworkModel.java

```diff
@@ -76,6 +76,7 @@ public class NetworkModel implements JsonRpcServiceObserver {
     public final LiveData<NetworkInfo> mDefaultNetwork;
     public final LiveData<List<NetworkInfo>> mPrimaryNetworks;
     public final LiveData<List<NetworkInfo>> mSecondaryNetworks;
+    public LiveData<NetworkLists> mNetworkLists;
```

Comment:
It can be `final` as well.

Reply:
Updated https://github.com/brave/brave-core/pull/19467#discussion_r1278958347